package.json

```diff
@@ -43,7 +43,7 @@
     "dayjs": "1.10.6",
     "formik": "2.2.9",
     "formik-antd": "2.0.1",
-    "next": "11.1.0",
+    "next": "11.0.1",
```

Comment:
Hi @cw1997 , you're correct. It is caused by the NextJS upgrade, so I revert it back to the former version.